faunadb-java-dsl/src/main/java/com/faunadb/client/query/Language.java

```diff
@@ -12,22 +12,25 @@
 import java.util.Map;
 
 /**
- * Methods modeling the FaunaDB query language. This class is intended to be statically imported into your code:
+ * The {@link Language} class contains static constructors for the FaunaDB Query Language expressions.
+ * This class is intended to be statically imported into your code with:
  *
  * {@code import static com.faunadb.client.query.Language.*;}
  *
- * Each of these methods constructs a {@link Expr}, which can then be composed with other methods to form complex
- * expressions.
+ * <p>
+ * Each method in the {@link Language} class constructs a new {@link Expr} instance.
+ * No computation is executed until the expression is evaluated by the FaunaDB server.
+ * </p>
  *
- * <b>Examples:</b>
+ * <b>Example:</b>
  * <pre>{@code
- *   Expr existsValue = Exists(Ref(Class("some_class"), "ref"));
+ *   Expr createUserExpr = Create(Class("user"), Obj("data", Obj(
+ *     "username", Value("bob"),
+ *     "password", Value("abc123"),
+ *   )));
```

Comment:
1.   I think we should have a simple description of what this code snippet is doing.  
2.   If this is creating a class should we not use the `CreateClass` 
3.  Should we not add @see https://fauna.com/documentation/reference/queryapi#create_class_params_object

Reply:
In this case, we're creating a new instance of the `user` class. I'll add a comment explaining the code sample.